dbtvault-dev/macros/materialisations/shared_helpers.sql

```diff
@@ -12,7 +12,7 @@
 
 {%- macro is_any_incremental() -%}
     {%- if dbtvault.is_vault_insert_by_period() or dbtvault.is_vault_insert_by_rank() or is_incremental() -%}
-        {%- do return(true) -%}
+        {{- return(true) -}}
```

Comment:
This needs to be `{%- do return() -%}`, please change this back

Reply:
Resolved